src/components/text-editor/examples/text-editor-height.tsx

```diff
@@ -7,28 +7,14 @@ import { Component, h } from '@stencil/core';
  * container element.
  *
  * However, you can constrain the text editor to never grow beyond a certain height,
- * using `--text-editor-max-height` property.
+ * by setting a fixed height the component itself; or alternatively by setting
+ * a fixed- or max-height on on the container element of the component.
  *
  * In this example, the maximum height is set to `15rem`, which means that:
  * 1. the editor will adjust itself to the content inside,
- * pushing out its container and making it taller, until it reaches `15rem`.
- * 1. and also the editor will try to fill the available surface area
- * when you manually resize the container,
- * (or some other element causes the container to grow taller), until it reaches `15rem`.
- *
- * :::important
- * Naturally, if the container has a fixed small height,
- * The editor which has a lot of content will grow taller than the container.
- * Depending on the `overflow` property of the container,
- * you may get undesired rendering, or parts of the content
- * can become invisible for the end-users.
- *
- * You can test this by resizing this example (which will give it a hardcoded
- * `width` and `height`), and then typing several lines of text!
- *
- * Therefore, you need to make sure that `--text-editor-max-height`
- * is set to a value which correlates with the hardcoded height of the container.
- * :::
```

Comment:
@adrianschmidt as you can see in this last commit, I have removed a bunch of stuff. 

What is left of it is what is proper implementation according to me. However, there are scenarios in which this fails, and there is no logically explanation for it. Like the recording below:

https://github.com/Lundalogik/lime-elements/assets/35954987/5d3fbcd4-5daf-4bb1-90ad-6e4e464dbbb1

Another weird scenario is where you:
1. remove the `resize` and `overflow` possibilities from the example, 
2. and instead give the host in the example a `max-height`.
3. type multi lines of text in the editor, and it grows/overflows out of its own container and there is no way to get it fixed.

My best guess is that ProesMirror is doing something to the container using javascript. I don't see any inlined-styles, dynamically applied. However, I'm quite sure that something is going on in the background, aligning or expanding the divs, that we can't see.

So if you also can't find a logic, we can drop the last commit and go with what I did up-to that point.
What do you say?

Reply:
Ah, I looked at the video, and it looks like the editor adapts to the container element again when you resize the container element, right? That's interesting… I wonder if that can be used in some way 🤔

Reply:
> This also makes me wonder whether we should enable end users to resize the height of the text editor or not?

Yes, we should. I _HATE_ it when some bloody developer thinks they know better than me how much space I'll need, and decides to prevent me from resizing the textarea I'm trying to write something in! 😤 I can't count the number of times I've used the dev-tools to change that behaviour on some web page I was using… 🤮

Reply:
If faced with the choice, I'd rather have the height adjustable by the user, and _not_ have it auto-expand based on content, then have it auto-expand, but then not be able to resize it manually.

Reply:
if the text-editor too should be resizable, then we will probably have even more problems with the UI 😅 
I'll update the branch now that their strange behaving menubar is gone, and see what surprises that one can bring

Reply:
> Ah, I looked at the video, and it looks like the editor adapts to the container element again when you resize the container element, right? That's interesting… I wonder if that can be used in some way 🤔

Yes, this is why I thought something more dynamic should be going on in the background